addon/components/google-recaptcha.js

```diff
@@ -0,0 +1,104 @@
+import Component from "@ember/component";
+import layout from "../templates/components/google-recaptcha";
+import { alias } from "@ember/object/computed";
+import { isNone } from "@ember/utils";
+import { later } from "@ember/runloop";
+import { merge } from "@ember/polyfills";
+import { isPresent } from "@ember/utils";
+import { next } from "@ember/runloop";
+
+export default Component.extend({
+  /**
+   * Component based off ember-g-recaptcha ember component:
+   * https://github.com/algonauti/ember-g-recaptcha
+  */
+  layout,
+  classNames: ["g-recaptcha"],
+  sitekey: "6LeJomQUAAAAABVGp6Xk3PUZGXNWaHo3t1D7mwF3", // public captcha key
+  tabindex: alias("tabIndex"),
+
+  renderReCaptcha() {
+    if (isNone(window.grecaptcha) || isNone(window.grecaptcha.render)) {
+      later(() => {
+        this.renderReCaptcha();
+      }, 500);
+    } else {
+      let container = this.element;
+      if (container) {
+        let properties = this.getProperties(
+          "sitekey",
+          "theme",
+          "type",
+          "size",
+          "tabindex",
+          "hl"
+        );
+        let parameters = merge(properties, {
```

Comment:
why are you doing a `merge`? All of these properties are scalar, which means they'll be passes by value, as-is into the object that results from `getProperties`.

You can just set new keys on the `properties` object for the callbacks:

```
properties.callback = bind(this, 'successCallback');
properties['expired-callback'] = bind(this, 'expiredCallback');
```